conf/modules/general.config

```diff
@@ -34,6 +37,9 @@ process {
     }
 
     withName: '.*:NALLO:FQCRS' {
+
+        ext.pref = { "${reads}" }
```

Comment:
I think this should be `ext.prefix` and on line 30 too :)

Reply:
Definitely, thanks!